runescape-client/src/main/java/Widget.java

```diff
@@ -1211,7 +1213,8 @@ public void method4525(int var1, int var2) {
       signature = "(ZI)Llc;",
       garbageValue = "-215938927"
    )
-   public SpritePixels method4562(boolean var1) {
+   @Import("getspriteunid")
```

Comment:
This should be renamed to something else. I only named it "getspriteunid" because I didn't know exactly what it should be named

Reply:
I renamed it to getWidgetSprite, is that better?